--- origsrc/libpaper-1.1.23/lib/Makefile.am	2006-01-07 15:17:55.000000000 -0600
+++ src/libpaper-1.1.23/lib/Makefile.am	2008-05-18 17:48:07.640625000 -0500
@@ -1,7 +1,7 @@
 
 lib_LTLIBRARIES = libpaper.la
 libpaper_la_SOURCES = dimen.c paper.c dimen.h paperspecs.h
-libpaper_la_LDFLAGS = -version-info 2:2:1
+libpaper_la_LDFLAGS = -version-info 2:2:1 -no-undefined
 libpaper_la_CPPFLAGS = -D_REENTRANT
 
 BUILT_SOURCES = paperspecs.h
